api/contact.py

```
@@ -0,0 +1,502 @@
+        success=True,
+        message="Thank you for your message! We'll get back to you within 24-48 hours.",
+        submission_id=submission_id
+    )
+
+
+@router.get("/submissions", response_model=SubmissionsResponse)
+async def get_submissions(
+    limit: int = Query(50, ge=1, le=200),
+    offset: int = Query(0, ge=0),
+    status_filter: Optional[str] = Query(None, regex="^(pending|resolved|spam)$"),
+    current_user = Depends(require_admin) if AUTH_AVAILABLE else None
```

Ruff (0.14.10)

Unused function argument: `current_user`

(ARG001)